trymito.io/pages/spreadsheet-automation.tsx

```diff
@@ -0,0 +1,334 @@
+                }
+              </div>
+
+            </div>
+          </section>
+
+          <section>
+            <div className={classNames(pageStyles.subsection, 'center')}>
+              <h2>
+                The Benefits of Automating Excel and Google Spreadsheets
+              </h2>
+            </div>
+            <div className={classNames(pageStyles.subsection, trifoldStyles.container)}>
+              <div className={classNames(pageStyles.subsection_column)}>
+                <div className={classNames(trifoldStyles.image_container, 'margin-top-8rem-mobile-only')}>
+                  <Image src={'/large_data.png'} alt='Large Data' width={1000} height={500}/>
+                </div>
+                <h3>
+                  Handle large datasets faster
+                </h3>
+                <p>
+                  Excel and Google Sheets aren&apos;t designed for modern data. Insert 1 million rows of data into Excel and you&apos;ll wait 10 minutes every time you update your analysis.
+                </p>
+                <p>
+                  Modern data teams use Python to analyze millions of rows of data in just a couple of seconds. 
+                </p>
+              </div>
+              <div className={classNames(pageStyles.subsection_column)}>
+                <div className={classNames(trifoldStyles.image_container, 'margin-top-8rem-mobile-only')}>
+                  <Image src={'/custom_imports.png'} alt='Custom Import' width={1000} height={500}/>
+                </div>
+                <h3>
+                  Use the full power of Python
+                </h3>
+                <p>
+                  Excel used to be the data collection, analysis, and report presentation tool. Today, modern data teams use Excel primarily as the output of analyses. 
+                </p>
+                <p>
+                  Automating reports in Python lets you use AI and ML to extract insights, connect directly to all of your data sources, and utilize template analyses built by your organization.
+                </p>
+              </div>
+              <div className={classNames(pageStyles.subsection_column)}>
+                <div className={classNames(trifoldStyles.image_container, 'margin-top-8rem-mobile-only')}>
+                  <Image src={'/code.png'} alt='Mito Generated Code' width={1000} height={500}/>
+                </div>
+                <h3>
+                  Reduce bugs and key person risk
+                </h3>
+                <p>
+                  Excel&apos;s greatest strength and weakness are the same -- it hides the analysis logic and encourages you focus on the data. 
+                </p>
+                <p>
+                  Because Excel reports lack structure, its easy to introduce bugs into your report and its nearly impossible to transfer responsibility for a large Excel file to someone else on your team. 
+                </p>
```

Comment:
```suggestion
                <p>
                  Because Excel reports lack structure, it's easy to introduce bugs into your report just by copying and pasting wrong. Moreover, it's nearly impossible to transfer responsibility for a large Excel file to someone else on your team. 
                </p>
                <p>
                  By transitioning to Python, you reduce the chance of off-by-one errors and make it much easier for processes to last as long as you need them.
                </p>
```